package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@itwin/reality-capture-main",
+  "version": "1.0.0",
+  "description": "This project contains iTwin API SDK for ContextCapture and RealityDataAnalysis as well as reality data API utils in typescript.",
+  "packageManager": "pnpm@7.14.0",
+  "scripts": {
+    "build": "pnpm -r --filter=@itwin/reality-capture run build && pnpm -r --filter=@itwin/reality-capture-node run build && pnpm -r --filter=@itwin/reality-capture-tests run build",
```

Comment:
shouldnt this build the entire ts side of this monorepo?

Reply:
building all packages and linting them